plugins/woocommerce/src/Internal/Admin/Settings/PaymentsProviders.php

```diff
@@ -592,7 +644,13 @@ public function get_extension_suggestion_by_id( string $id ): ?array {
 	 * @return ?array The payment extension suggestion details, or null if not found.
 	 */
 	public function get_extension_suggestion_by_plugin_slug( string $slug, string $country_code = '' ): ?array {
-		return $this->extension_suggestions->get_by_plugin_slug( $slug, $country_code, Payments::SUGGESTIONS_CONTEXT );
+		$suggestion = $this->extension_suggestions->get_by_plugin_slug( $slug, $country_code, Payments::SUGGESTIONS_CONTEXT );
```

Comment:
To ensure consistency, always enhance the suggestion data, the same way to provide it in the providers list.